This patch is only needed by the 'imx8mp_qcacld2_wifi' driver.

This patch makes 'driver_deferred_probe_enable' accessible from the
outside so that we can set at the appropriate time. This is needed
because the way the 'qcacld-2.0' driver is currently enabled requires
items from the deferred list to be processed even before the late
initcall stage is reach where is type of deferred probing is normally
performed.

--- a/drivers/base/dd.c
+++ b/drivers/base/dd.c
@@ -153,7 +153,7 @@ void driver_deferred_probe_del(struct device *dev)
 	mutex_unlock(&deferred_probe_mutex);
 }
 
-static bool driver_deferred_probe_enable;
+bool driver_deferred_probe_enable;
 /**
  * driver_deferred_probe_trigger() - Kick off re-probing deferred devices
  *

Increase the DMA buffer to 64 KiB because at times someone tries to send
a bundle that is much larger (so far around 34 KiB at most were observed)
and this is then dropped by the driver.
--- a/qca/CORE/SERVICES/HIF/sdio/linux/native_sdio/include/hif_internal.h
+++ b/qca/CORE/SERVICES/HIF/sdio/linux/native_sdio/include/hif_internal.h
@@ -143,7 +143,7 @@ struct hif_device {
 #endif
 };
 
-#define HIF_DMA_BUFFER_SIZE (4 * 1024)
+#define HIF_DMA_BUFFER_SIZE (16 * 4 * 1024)
 #define CMD53_FIXED_ADDRESS 1
 #define CMD53_INCR_ADDRESS  2
 
